#!/usr/bin/python3


def raise_exception():
    result = 10 / "hello"
